Resources/Prototypes/_Goobstation/Entities/Objects/Devices/Electronics/misc_signallers.yml

```
@@ -0,0 +1,71 @@
+
+- type: entity
+  parent: BaseItem
+  id: MiscSignallerBase
+  name: "misc signaller base"
+  abstract: true
+  components:
+  - type: MiscSignaller
+  - type: StaticPrice
+    price: 40
+  - type: DeviceLinkSource
+    ports: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)